gast/ast3.py

```diff
@@ -88,6 +88,32 @@ def visit_ExceptHandler(self, node):
         else:
             return self.generic_visit(node)
 
+    if sys.version_info.minor < 5:
+
+        def visit_Call(self, node):
+            self.generic_visit(node)
+            new_node = ast.Call(
+                func=self._visit(node.func),
+                args=self._visit(node.args),
+                keywords=self._visit(node.keywords),
+                starargs=[],
+                kwargs=[],
```

Comment:
Here it is not a star argument but an optional argument so empty should be "None" instead of "[]". This is was generated your error on travis